lib/services/settlement/app/src/main/java/com/amazon/settlement/service/SubscriberService.java

```diff
@@ -0,0 +1,84 @@
+package com.amazon.settlement.service;
+
+import com.amazon.settlement.model.input.Settlement;
+import com.amazon.settlement.model.output.Detail;
+import com.amazon.settlement.model.output.ReturnValue;
+import com.amazon.settlement.repository.SettlementRepository;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.cloud.aws.messaging.listener.SqsMessageDeletionPolicy;
+import org.springframework.cloud.aws.messaging.listener.annotation.SqsListener;
+import org.springframework.messaging.handler.annotation.Header;
+import org.springframework.stereotype.Component;
+import software.amazon.awssdk.services.eventbridge.EventBridgeClient;
+import software.amazon.awssdk.services.eventbridge.model.PutEventsRequest;
+import software.amazon.awssdk.services.eventbridge.model.PutEventsRequestEntry;
+import software.amazon.awssdk.services.eventbridge.model.PutEventsResponse;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Slf4j
+@Component
+public class SubscriberService {
```

Comment:
I will talk to you about what I have been thinking when we elaborate this for the blog